tests/integration/modules/test_vsphere.py

```diff
@@ -52,3 +52,13 @@ def test_list_ssds(service_instance):
     for host_name in ret:
         for ssd in ret[host_name]:
             assert isinstance(ssd, str)
+
+
+def test_list_non_ssds(service_instance):
+    ret = vsphere.list_ssds(
```

Comment:
didn't the last PR have a way to return SSDs and non-SSDs so

- Why have a separate function to get non_ssds, see  _get_host_disks, justr don't see need for helper function
- shouldn't this be vsphere.list_non_ssds

Reply:
Doh, good catch. Thank you!